<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queueDialog</class>
 <widget class="QDialog" name="queueDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Queue Management</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="closeButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>441</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="slidersLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
